mlops/parallelm/mlops/mlops_metrics.py

```diff
@@ -128,3 +128,16 @@ def mean_squared_error(y_true, y_pred, sample_weight=None, multioutput="uniform_
                                                  multioutput=multioutput)
 
         mlops.set_stat(RegressionMetrics.MEAN_SQUARED_ERROR, data=mse)
+
+    @staticmethod
+    def mean_squared_log_error(y_true, y_pred, sample_weight=None, multioutput="uniform_average"):
+        # need to import only on run time.
```

Comment:
What about documentation for the method using a doc string.. this applies for all PR's

Reply:
yes. and it is added to all in beautify patch :)